sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -333,7 +349,7 @@
     public Mono<ConfigurationSetting> deleteSetting(String key, String label) {
         return withContext(
             context -> deleteSetting(new ConfigurationSetting().setKey(key).setLabel(label), false, context))
-            .flatMap(response -> Mono.justOrEmpty(response.getValue()));
+            .map(response -> response.getValue());
     }
```

Comment:
This needs to use justOrEmpty as the response value can be null which is an illegal value in a Reactor stream.